event_model/__main__.py

```diff
@@ -0,0 +1,9 @@
+from event_model._version import __version__
+import argparse
+
+parser = argparse.ArgumentParser(description="For getting information about the event-model installation.")
+version_arg = parser.add_argument("--version", action="version", version=__version__)
+
+if __name__ == "__main__":
+    if version_arg.version:
+        print(version_arg.version) 
```

Comment:
This looks different from https://github.com/DiamondLightSource/python3-pip-skeleton/blob/main/src/python3_pip_skeleton/__main__.py

Is it different for a reason? If so should we push it upstream too?

Reply:
Nope, I think I was accounting for a problem in my build no longer present.